base/operators.jl

```diff
@@ -871,26 +871,54 @@ julia> fs = [
 julia> ∘(fs...)(3)
 3.0
 ```
+See also [Base.ComposedFunction](@ref).
```

Comment:
I guess we need backquotes? Let's see...

```suggestion
See also [`Base.ComposedFunction`](@ref).
```